administration/src/mui-modules/application/forms/StepCardTypeForm.tsx

```diff
@@ -105,15 +128,23 @@ const StepCardTypeForm: Form<State, Options, ValidatedInput, AdditionalProps> =
         <CustomDivider />
         <Typography>
           Die Ehrenamtskarte ist als physische Karte und als digitale Version für Ihr Smartphone oder Tablet erhältlich.
-          Hier können Sie wählen, ob Sie neben der physischen auch kostenfrei die digitale Ehrenamtskarte beantragen
-          möchten.
+          Hier können Sie auswählen welchen Kartentyp Sie beantragen möchten.
```

Comment:
Sounds a bit as if the user would have to choose between one or the other? Maybe:
```suggestion
          Hier können Sie auswählen, welche Kartentypen Sie beantragen möchten.
```